src/main/kotlin/com/mparticle/kits/RoktKit.kt

```diff
@@ -318,6 +320,40 @@ class RoktKit : KitIntegration(), CommerceListener, IdentityListener, RoktListen
         }
     }
 
+    private fun verifyHashedEmail(attributes: MutableMap<String, String>?) {
+        if (attributes == null) return
+
+        val emailKey = MParticle.IdentityType.Email.name.lowercase()
+        val otherKey = MParticle.IdentityType.Other.name.lowercase()
+        val emailShaKey = "emailsha256"
+
+        val emailShaValue = attributes.entries.find { it.key.equals(emailShaKey, ignoreCase = true) }?.value
+        val otherValue = attributes.entries.find { it.key.equals(otherKey, ignoreCase = true) }?.value
```

Comment:
Is there a reason we're still getting otherValue here? I updated the code that adds the identities to the attributes so that the other identity gets mapped to the 'emailsha256' key. That way if the customer passes in an 'other' attribute it doesn't get overwritten.